test/e2e/capi_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
We can remove this line, it's a weird issue where Go 1.17 will add it when generating files.